packages/react-instantsearch/src/connectors/connectRefinementList.js

```diff
@@ -100,9 +100,23 @@ export default createConnector({
         isRefined: v.isRefined,
       }));
 
+    let facetValues;
+    if (facetValuesResults) {
+      facetValues = facetValuesResults[attributeName]
+        .slice(0, limit).map(
+          v => ({
+            label: v.value,
+            value: getValue(v.value, props, searchState),
+            _highlightResult: {label: {value: v.highlighted}},
+            count: v.count,
+            isRefined: v.isRefined,
+          }));
+    }
+
     return {
       items,
       currentRefinement: getCurrentRefinement(props, searchState),
+      facetValues,
```

Comment:
To do that we need to add the SFFV query inside the search state. 
Currently the SFFV query is handled directly inside the RefinementList widget by using state. It means that you can't share an url that will bring you to an InstantSearch page with a SFFV displayed. Do we want that?

Also adding it to the search state brings new questions. Basically I assumed we would want a structure like:
```
{
   refinementList: {category: {items: ['Table'], query: 'Ch'}}
}
```

But then when forwarding the `searchState` to the user, how could he easily know that it's not necessary to update the whole <InstantSearch/> (that would trigger a new search for nothing)?